### ISSUE TITLE
f128::MAX and NaN

### ISSUE THREAD

**Issue by frgomes** [jkarns275/f128#28]
Why
```rust
println!("{:?}", f128::MAX);
```
prints ``-nan``?

Bug or feature?

**Comment by frgomes** [comment]
This is some test code, which fails:

```rust
#[cfg(test)]
mod tests {
    use ::f128::*;
    use num_traits::Float;
    use super::*;
    
    #[test]
    fn max_is_not_nan() {
        assert!(! f128::MAX.is_nan()); // see: https://github.com/jkarns275/f128/issues/28
    }
}
```

Similar code for ``f64`` pass as expected.

**Comment by frgomes** [comment]
I see that the code agrees with [quadruple precision specification in IEEE-754](https://en.wikipedia.org/wiki/Quadruple-precision_floating-point_format):
```rust
    #[cfg(target_endian = "little")]
    pub const MAX: f128 = f128([
        0x7f, 0xfe, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
        0xff,
    ]);
    #[cfg(target_endian = "big")]
    pub const MAX: f128 = f128([
        0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfe,
        0x7f,
    ]);
```
So, I'm puzzled about the root cause for this behaviour.

**Comment by jkarns275** [comment]
The root cause is incorrect byte order - the little and big endian versions of those constants need to be swapped. Going to add some more tests for this sort of thing and push a fix. I believe f128::MIN was also in the wrong order.

**Comment by jkarns275** [comment]
This was fixed by 3a3b834afc3db9e8e354d2389f0c5e75cbef8c2f. I'll push a new version of the crate shortly
